Task 4 by erkinchi

erkinchi opened the pull request.

erkinchi committed:
Task 4 by erkinchi

IneryTestnet commented:
Your solution is not valid. Please, feel free to use our js library or Python library on our Github profile. Read more in README.md. Please do not use AI for generating the code or something like that, because it is not correct.

erkinchi committed:
Task 4 (New)

My previous solution is valid. I did test again. And I discussed this issue with Lazar on discord channel. It supposed to be tested again.. I also wrote this new solution with Python. 

Thank you

erkinchi commented:
My previous solution is valid. I did test again. And I discussed this issue with Lazar on discord channel. It supposed to be tested again.. I also wrote this new solution with Python. Thank you
